Justfile

```diff
@@ -84,7 +84,7 @@ test-e2e-tier2: pnpm-install deploy-mina-indexer && shutdown-mina-indexer
   CYPRESS_tags="@tier2" \
   GRAPHQL_URL="$GRAPHQL_URL" \
   REST_URL="$REST_URL" \
-  time ruby ./ops/manage-processes.rb \
+  ruby ./ops/manage-processes.rb \
```

Comment:
(non-blocking, question): No more `time`??

Reply:
I couldn't kill the subprocesses. But I think the other PR handles it, so i'll put it back in.

Reply:
```suggestion
  time ruby ./ops/manage-processes.rb \
```